Lesson14/fikriye/script.js

```diff
@@ -0,0 +1,470 @@
+   - Log "Secure connection" if true, otherwise "Unsecure connection".
+*/
+function checkProtocol(url){
+  url = url.toLowerCase ();
+  if (url.startsWith("https")){
+    console.log("Secure connection" )
+  }
+  else{
+    console.log("Unsecure connection");
+  }
+
+}
+checkProtocol("https://asdjhsfg");
+
+/*
+12. Switch: Day of the Week
+   - Define a function `getDayOfWeek(num)` that uses a switch statement:
+     1 -> "Monday"
+     2 -> "Tuesday"
+     ...
+     7 -> "Sunday"
+     - Log the matched day or "Invalid day" if out of range.
+*/
+function getDayOfWeek(num){
+  switch (num) {
+    case 1:
+        console.log("Monday");
+        break;
+    case 2:
+        console.log("Tuesday");
+        break;
+    case 3:
+        console.log("Wednesday");
+        break;
+    case 4:
+        console.log("Thursday");
+        break;
+    case 5:
+        console.log("Friday");
+        break;
+    case 6:
+        console.log("Saturday");
+        break;
+    case 7:
+        console.log("Sunday");
+        break;
+    default:
+        console.log("Invalid day");
+}
+}
+getDayOfWeek(5);
+
+/*
+13. Repeat a String
+   - Define a function `repeatWord(word, times)` that uses the .repeat() method
+     to repeat `word` `times` times.
+   - Log the repeated result.
+*/
+function repeatWord(word,times){
+  const sonuc=word.repeat(times);
+  console.log(sonuc);
+}
+repeatWord("merhaba",5);
+/*
+14. Replace Substring
+   - Define a function `censorWord(sentence, target)` that replaces `target`
+     with "****" (use .replaceAll() or multiple .replace()).
+   - Log the censored sentence.
+*/
+function censorWord(sentence, target){
+  const censored = sentence.replaceAll(target, "****");
+    console.log(censored);
+}
+     censorWord("Hasta olmak çok kötü çok çok kötü","kötü");
+/*
+15. Check First Character (if-else)
+   - Define a function `startsWithA(str)` that checks if the string starts with 'A'
+     (use .charAt(0) or [0]).
+   - Log "Starts with A" or "Does not start with A".
+*/
+function startsWithA(str){
+  if (str[0] === 'A'){
+    console.log("Starts with A");} 
+  else {
+    console.log("Does not start with A");
+  }
+}
+startsWithA("Ayva");
+startsWithA("muz");
+
+/*
+16. Slice Last N Characters
+   - Define a function `sliceLastN(text, n)` that uses .slice(-n) to extract
+     the last `n` characters of `text`.
+   - Log the result.
+*/
+function sliceLastN(text,n){
+const sonuc=text.slice(-n);
+console.log(sonuc);
+}
+sliceLastN("TimUraz" ,4);
+/*
+17. Switch: Grade Checker
+   - Define a function `gradeChecker(score)` that uses a switch (or if-else chain):
+     90+ -> "A"
+     80-89 -> "B"
+     70-79 -> "C"
+     60-69 -> "D"
+     below 60 -> "F"
+   - Log the grade.
+*/
+function gradeChecker(score){
+  if (score >= 90) {
+    grade = "A";
+} else if (score >= 80) {
+    grade = "B";
+} else if (score >= 70) {
+    grade = "C";
+} else if (score >= 60) {
+    grade = "D";
+} else {
+    grade = "F";
+}
+}
+console.log(72);
```

Comment:
You're logging the number 72, not the grade. Update like this:
```suggestion
console.log(gradeChecker(72));
```